sdks/python/apache_beam/yaml/yaml_provider.py

```diff
@@ -429,7 +441,9 @@ def parse_schema(spec):
           ],
           id=str(uuid.uuid4()))
 
-    named_tuple = schemas.named_tuple_from_schema(parse_schema(args))
+    if 'schema' not in args.keys():
+      raise ValueError("WithSchema transform missing required 'schema' tag.")
+    named_tuple = schemas.named_tuple_from_schema(parse_schema(args['schema']))
```

Comment:
@robertwb Should I remove this now that we have the `config` tag?

Reply:
The schema needs to be in the config now.